modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -5,31 +5,46 @@
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
 // * installing/installing_openstack/installing-openstack-user.adoc
 
+ifeval::["{context}" == "installing-openstack-user"]
+:osp-user:
```

Comment:
Not going to lie, :osp-upi:, like in the next module, makes more sense to scan.

Reply:
Yeah, this was an experiment to cut down on ifevals. Might not persist through the next release.